bot/extensions/activities.py

```diff
@@ -0,0 +1,37 @@
+from discord.ext.commands import Cog, command
+import os
+import requests
+import json
+import discord
+
+class activities(Cog, name="Activities",description="Commands regarding Discord Activities in VC" ):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @command(name='youtube', help='Will enable Youtube Together in VC', usage='youtube')
+    async def youtube(self, ctx):
+        try:
+            channel = ctx.author.voice.channel
+            url = f"https://discord.com/api/v9/channels/{channel.id}/invites"
+            p = {
+                'max_age': 86400,
+                'max_uses': 0,
+                'target_application_id': '755600276941176913', 
+                'target_type': 2,
+                'temporary': False,
+                        'validate': None
+            }
+            headers={'content-type': 'application/json','Authorization': f"Bot {os.getenv('DISCORD_TOKEN')}"}
+            r=requests.post(url, data=json.dumps(p), headers=headers)
+            
+            embed = discord.Embed(title="Youtube Together",colour=discord.Colour(0xFF0000))
+            embed.description = f"[Click Here to Join Youtube Together in the VC](https://discord.com/invite/{r.json()['code']})"
+            await ctx.send(embed=embed)
+        except AttributeError:
+             await ctx.reply("You Are not in a VC")
+        except Exception as e:
```

Comment:
Sometimes it still gives error even if the bot has permission to create invite and I don't know why. If you have other suggestion, pls tell.

Reply:
Frankly is difficult to know what kind of error you get but you should be able to see exactly the type or error by looking in a server logs when it happens. It's gonna say something like:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can only concatenate str (not "int") to str
```
In this example, the error was caused by doing `"5" + 5` and as you know you can't add a **string** type with an **integer** type, thus and error occurs. You can spot the error type at the end where it's says `TypeError`. You can find the cause of the exception by looking there in the logs. 

If an exception isn't thrown and it simply shows a command error it will say the cause of the error and with that you can trace it back. If it happens, it will show something like that `Error: Command "test" is not found. Issued by Malassi#0410` in yellow. 

If it's neither those case or you simply can't find the error you can contact me on discord and we will check that in VC or in text chat.

Reply:
Even I forgot the error, but I remember that sometimes I got an error. I don't remeber the exact condition & how to replicate the error

Reply:
Ok I'll check this out soon.

Reply:
👍